tests/draft2019-06/additionalItems.json

```diff
@@ -6,12 +6,176 @@
       {
         "description": "additional items match schema",
         "data": [null, 2, 3, 4],
-        "valid": true
+        "valid": true,
+        "output": {
+          "basic": {
+            "valid": true,
+            "keywordLocation": "#",
```

Comment:
These extra properties in the `basic` output are a serialization artifact of my implementation.  Probably should remove them.  I'll figure out if there's a way to do that easily.

Reply:
No... this is right.  If there are more than one error, then those errors are collected in a list, but a single error is just represented at the root.